Makefile

```diff
@@ -1,50 +1,98 @@
+{%- comment -%} dynamic make targets {%- endcomment -%}
+{%- assign target_dir = "./target" -%}
+{%- assign lib_dir = "./native/lib" -%}
+{%- assign bin_dir = "./native/bin" -%}
+{%- assign aarch64-linux-android-debug = "aarch64-linux-android,Android,debug" | split: "|" -%}
+{%- assign aarch64-linux-android-release = "aarch64-linux-android,Android,release" | split: "|" -%}
+{%- assign armv7-linux-androideabi-debug = "armv7-linux-androideabi,Android,debug" | split: "|" -%}
```

Comment:
Not necessarily an issue, but what do the `split: "|"`s here and below accomplish? I don't see any `|`s in the string.

Reply:
It puts the string inside an array so I could concat it with the other arrays. The other reason is its becoming long horizontally if I put multiple (or all) targets in one line. There is no way in liquid to initialize arrays...